### PR TITLE
Clamp max attribute name length to [63, 2047] and comment

### PR DESCRIPTION
**Pull request by willemv**
Let's not trust the driver too much.

### CODE REVIEW

**Comment by est31** [comment]
Can you add a comment why you are doing a max? Also, it'd be helpful to add a comment as I described in #1898.

**Comment by willemv** [comment]
Let's also add an upper bound instead of only a lower bound.

**Comment by willemv** [comment]
> 
> 
> Can you add a comment why you are doing a max? Also, it'd be helpful to add a comment as I described in #1898.

will do

File: src/program/reflection.rs [hunk 1]
```diff
@@ -345,6 +346,9 @@ pub unsafe fn reflect_attributes(ctxt: &mut CommandContext<'_>, program: Handle)
                                                 &mut attr_name_len_max);
             }
         };
+        //let's not trust the driver too much, and clamp the max_len to [63, 2047]
```
Comment (est31) [inline src/program/reflection.rs:349]: Can you add a space between // and the text? This is more commonly used in the codebase.

**Review by est31 — CHANGES_REQUESTED** [review]

File: src/program/reflection.rs (hunk 1 above)
Comment (willemv) [reply src/program/reflection.rs]: Done!

**Review by est31 — APPROVED** [review]
Thanks!

**Pull request merged**